### DISCUSSION TITLE
Moon Universe

### DISCUSSION THREAD

**Discussion by ShivamAgarwal-code** [planetarium/NineChronicles.Mods#116] [Ideas]
My idea for mod idea is Moon Universe. It is a A web3, cyber-fantasy, open-world RPG optimised for mobile. Welcome to a unique top-down isometric 2.5D RPG, where players can explore a city map, engage in a battle arena and take on relentless alien critters. This project stands out with it's integration of the blockchain, enabling players to upgrade in-game items using Solana, Bonk and USDC tokens as well as adventure, fish, trade and battle! Moon Universe merges cutting-edge game design with the SPL 404 standard on Solana, enabling seamless integration of coins and NFTs in a unified economy. This creates an immersive, player-driven experience where in-game achievements translate to real-world value. The game’s mobile-first approach ensures smooth, accessible gameplay for a wider audience, bridging the gap between mainstream and blockchain gaming.
![image](https://github.com/user-attachments/assets/d0951672-e636-45f9-87b9-5b0a1be0f7e1)
![image](https://github.com/user-attachments/assets/3a239569-63aa-41fa-9706-9b65924808e4)

**Comment by Atralupus** [comment]
I'm sorry, but mods related to 9c would be a better fit. Thank you for sharing your thoughts!
